src/Entity/Carrinho.php

```diff
@@ -96,7 +94,7 @@ public function setStatus(string $status): static
 
     public function getValorTotal(): ?int
     {
-        return $this->valorTotal;
+        return $this->valorTotal / 100;
```

Comment:
a assinatura do retorno muda para ?float, esse bcdiv é para garantir que o processador da máquina não vai calcular os decimais errados...

```suggestion
        return (string) bcdiv($this->valorTotal, '100', 2);
```